apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -1944,15 +1944,20 @@ private static CareTeamComponent addCareTeamPractitioner(
     CareTeamComponent careTeamEntry =
         eob.getCareTeam().stream()
             .filter(ctc -> ctc.getProvider().hasIdentifier())
+            .filter(ctc -> ctc.hasRole())
             .filter(
                 ctc ->
-                    type.getSystem().equals(ctc.getProvider().getIdentifier().getSystem())
-                        && practitionerIdValue.equals(ctc.getProvider().getIdentifier().getValue()))
-            .filter(ctc -> ctc.hasRole())
+                    ctc.getProvider().getIdentifier().getType().getCoding().stream()
+                            .anyMatch(c -> c.getSystem().equalsIgnoreCase(type.getSystem()))
+                        && ctc.getProvider().getIdentifier().getType().getCoding().stream()
+                            .anyMatch(c -> c.getCode().equalsIgnoreCase(type.name()))
```

Comment:
I think these two checks could be merged into a less verbose 
```
 ctc.getProvider().getIdentifier().getType().getCoding().stream()
                            .anyMatch(c -> c.getSystem().equalsIgnoreCase(type.getSystem()) && c.getCode().equalsIgnoreCase(type.name()))
```
Since they are both working on the codings and checking for things in the same object

Reply:
done